content/docs/user-guide/networking/multiplayer/autocomponents.md

```diff
@@ -9,47 +9,132 @@ weight: 400
 
+
+### Network properties
+
+Network properties state information about the component that gets replicated between hosts. 
+They have two important fields: `ReplicateFrom` and `ReplicateTo`. 
+Together, these fields define which role can replicate to another specific role.
+You can only replicate property values *from* Authority and Autonomous roles. 
+A `ReplicateFrom` Authority relationship creates a server-authority model, which helps to ensure that you don't accidentally replicate from an unprivileged client. 
+Properties can be replicated *to* any role, because all participants in the session may need information from any other participant.
+
+The following are four types of networking properties with a valid "from and to" relationship and their possible use cases:
+
+- **Authority-to-Client**: Handles client, or "simulated", properties.
+
+- **Authority-to-Autonomous**: Handles autonomous-only properties.
+
+- **Authority-to-Server**: Handles host migration.
+
+- **Autonomous-to-Authority**: Gathers information about client metrics, such as monitoring the health of clients.
+
+For networking properties that replicate from Authority, a replication hierarchy applies. The replicate-to rules trickle up the hierarchy in the following way: An Authority-to-Client replication also replicates to Autonomous and Server roles. An Authority-to-Autonomous replication also replicates to the Server role. Finally, an Authority-to-Server replication only replicates to the Server. This behavioral hierarchy ensures that if the Authority ever migrates to the other server, then the other server has the right property information.
+
+The `NetworkProperty` tag has the following properties:
+
+| Property | Description | Values |
+| - | - | - |
+| Name | Name of the network property. | `string` |
+| Description | Describes the function of the network property. | `string` |
+| Type | The property's type must be a valid C++ type or class name. | `string` |
+| Init | The initial value of the property. | `string` |
+| ReplicateFrom | Tells the network which role can send information about the changes that were made to this property. | Authority, Autonomous | 
+| ReplicateTo | Tells the network which role can receive information about the changes that were made to this property. | Server, Authority, Autonomous, Client | 
+| Container | The type of holder object that stores this archetype property. | `Object`, `Array`, `Vector` |
+| Count | The number of elements, when `Container` is `Array` or `Vector`. Must be an integer value or an integer type variable. | `string` |
+| IsPublic | If `true`, this property's access modifier is set to `public`, and is accessible from outside of the class.  | `bool` |
+| IsRewindable |  | `bool` |
+| IsPredictable |  | `bool` |
+| ExposeToEditor | If `true`, this property is accessible in the O3DE Editor. | `bool` |
+| ExposeToScript | If `true`, this property is accessibly via script.  | `bool` |
+| GenerateEventBindings |  | `bool` |
```

Comment:
If true callbacks (see AZ::Events) will be triggered anytime the value of this network property is changed.